src/pages/faretype.tsx

```diff
@@ -1,79 +1,106 @@
-                                    id="faretype-period"
-                                    name="faretype"
-                                    type="radio"
-                                    value="period"
-                                />
-                                <label className="govuk-label govuk-radios__label" htmlFor="faretype-period">
-                                    Period Tickets
-                                </label>
-                            </div>
-                            <div className="govuk-radios__item">
-                                <input
-                                    className="govuk-radios__input"
-                                    id="faretype-return"
-                                    name="faretype"
-                                    type="radio"
-                                    value="return"
-                                    disabled
-                                    aria-disabled="true"
-                                />
-                                <label className="govuk-label govuk-radios__label" htmlFor="faretype-return">
-                                    Return
-                                </label>
+type FareTypeProps = {
+    errors: ErrorInfo[];
+};
+
+const FareType = ({ errors = [] }: FareTypeProps): ReactElement => {
+    return (
+        <Layout title={title} description={description}>
+            <main className="govuk-main-wrapper app-main-class" id="main-content" role="main">
+                <form action="/api/fareType" method="post">
+                    <ErrorSummary errorHref="#radio-buttons" errors={errors} />
+                    <div className={`govuk-form-group ${errors.length > 0 ? 'govuk-form-group--error' : ''}`}>
+                        <fieldset className="govuk-fieldset" aria-describedby="fareType-page-heading">
+                            <legend className="govuk-fieldset__legend govuk-fieldset__legend--xl">
+                                <h1 className="govuk-fieldset__heading" id="fareType-page-heading">
+                                    What type of fare would you like to provide?
+                                </h1>
+                                {errors.length > 0 && (
+                                    <span id="fareType-error" className="govuk-error-message error-message-padding">
+                                        <span>{errors[0].errorMessage}</span>
+                                    </span>
+                                )}
+                            </legend>
+                            <div className="govuk-radios" id="radio-buttons">
+                                <div className="govuk-radios__item">
+                                    <input
+                                        className={`govuk-radios__input ${
+                                            errors.length > 0 ? 'govuk-input--error' : ''
+                                        } `}
```

Comment:
How come this className is different to the other radio buttons?

Reply:
Another good catch, changed the className for all radio buttons (other pages also).